cmd/lotus-shed/verifreg.go

```diff
@@ -409,3 +413,115 @@ var verifRegCheckVerifierCmd = &cli.Command{
 		return nil
 	},
 }
+
+var verifRegRemoveVerifiedClientDataCapCmd = &cli.Command{
+	Name:      "remove-verified-client-data-cap",
+	Usage:     "Remove data cap from verified client",
+	ArgsUsage: "<message sender> <verified client ID> <allowance to remove> <verifier 1 ID> <verifier 1 signature> <verifier 2 ID> <verifier 2 signature>",
+	Action: func(cctx *cli.Context) error {
+		if cctx.Args().Len() != 7 {
+			return fmt.Errorf("must specify seven arguments: sender, client, allowance to remove, verifier 1 ID, verifier 1 signature, verifier 2 ID, verifier 2 signature")
+		}
+
+		srv, err := lcli.GetFullNodeServices(cctx)
+		if err != nil {
+			return err
+		}
+		defer srv.Close() //nolint:errcheck
+
+		api := srv.FullNodeAPI()
+		ctx := lcli.ReqContext(cctx)
+
+		sender, err := address.NewFromString(cctx.Args().Get(0))
+		if err != nil {
+			return err
+		}
+
+		client, err := address.NewFromString(cctx.Args().Get(1))
+		if err != nil {
+			return err
+		}
+
+		allowanceToRemove, err := types.BigFromString(cctx.Args().Get(2))
+		if err != nil {
+			return err
+		}
+
+		verifier1Addr, err := address.NewFromString(cctx.Args().Get(3))
```

Comment:
let's resolve all these things (so that users can specify the robust addresses if they want)

Reply:
(so the 2 verifiers and the clients)

Reply:
Robust addresses will get resolved by the function that processes this message.